joomla/joomla-cms
# [4.3] Fix dynamic properties warnings on Multilinual Sample Data plugin

Pull Request for Issue # .

### Summary of Changes
This PR fixes Creation of dynamic property warnings on Multilinual Sample Data plugin on PHP 8.2. We assign return values of two methods two none-existent properties and PHP 8.2 throws warnings. I found this issue while testing PR https://github.com/joomla/joomla-cms/pull/40216

As these properties are not used at all inside the plugin (you can scan code to confirm), we can just return the values directly.

### Testing Instructions
1. Use Joomla 4.3 night build on PHP 8.2
2. Set Error Reporting to Maximum
3. Install an extra language (to setup multilingual website)
4. Install Multilinual Sample Data

Code review should also be enough.

### Actual result BEFORE applying this Pull Request
There is an error with set up sample data. The process could not be finished


### Expected result AFTER applying this Pull Request
No error. Sample data property installed.


### Link to documentations
Please select:
